src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java

```diff
@@ -448,11 +447,9 @@ protected Object invokeDescribable(String symbol, Object _args) {
             // where this UninstantiatedDescribable is ultimately used, the symbol
             // might be resolved with a specific type.
 
-            // we are returning the NamedArgsAndClosure instead of the UninstantiatedDescribable in order to preserve
-            // the discovered interpolated strings that are stored in the NamedArgsAndClosure.
-            args.uninstantiatedDescribable = ud;
-            return args;
+            return new UninstantiatedDescribableWithInterpolation(symbol, null, args.namedArgs, args.interpolatedStrings);
         } else {
+            UninstantiatedDescribable ud = new UninstantiatedDescribable(symbol, null, args.namedArgs);
```

Comment:
Are you intentionally not adding the interpolated strings here? Does it not matter because we will scan everything in `invokeStep` in this case anyway?

Reply:
Yes, in this case the interpolated strings were scanned already in the `NamedArgsAndClosure` here: https://github.com/jenkinsci/workflow-cps-plugin/blob/7f00f8c8d40f5a3e75abc1a4a1ca14f13ab929dd/src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java#L436
and then used in `invokeStep` as a new `NamedArgsAndClosure`. I guess I could have left it outside of the `if-else` statement, I just reverted it to match a bit closer to the old code.

Reply:
Ok, the way you have it seems fine to me, just wanted to make sure it was intentional.